frontend/src/modules/search/components/FiltersPanel.js

```diff
@@ -86,6 +86,7 @@ export class FiltersPanelBase extends React.Component<Props, State> {
         this.setState({
             visible: false,
         });
+        this.props.resetFilters();
```

Comment:
Nice, this fixes the bug for the use case described in the bug:
https://bugzilla.mozilla.org/show_bug.cgi?id=1344120#c9

However, it introduces another problem: If you apply some filters first (e.g. Missing) and then try to reproduce the bug, the filters will be reset to the default state (nothing selected) instead of the previous state.

See how to solve the bug properly:
https://bugzilla.mozilla.org/show_bug.cgi?id=1344120#c13